### DISCUSSION TITLE
CLAASP installation

### DISCUSSION THREAD

**Discussion by Cybro0** [Crypto-TII/claasp#126] [Q&A]
While installing CLAASP in a system. Im getting this error while importing a cipher in sage terminal. How to resolve this 
![image](https://github.com/Crypto-TII/claasp/assets/143835501/796db733-795b-4ca2-8f6c-25460cf4f3f8)

**Comment by peacker** [comment]
Hi Cybro0, have you tried to use CLAASP through its docker image? We recommend this approach at the moment. If you really prefer to avoid Docker, it seems you need to have the same version of numpy as the one indicated in the requirements file. If neither of the above does not work, please let us know and we can assist further.

**Reply by Cybro0** [reply]
Yes, I installed it through docker image itself.

**Reply by juaninf** [reply]
@Cybro0, thanks for pointing out this issue; now it is solved.

**Reply by Cybro0** [reply]
No, Im still facing the issue.

**Reply by juaninf** [reply]
Could you please send here the output after updating the develop branch (with pull) and running `make rundocker` ?

**Reply by juaninf** [reply]
Also, we can try to do a video meeting so in this way, you can share your screen and we can improve our communication

**Reply by Cybro0** [reply]
Yeah, its working now. Thank you.
